File: SkyEditor.SaveEditor/MysteryDungeon/Explorers/IExplorersStoredPokemon.vb
Namespace MysteryDungeon.Explorers
    Public Interface IExplorersStoredPokemon
        Property Level As Byte
        Property ID As Integer
        Property IsFemale As Boolean
        Property MetAt As Integer
        Property MetFloor As Integer
        Property IQ As Integer
        Property HP As Integer
        Property Attack As Byte
        Property Defense As Byte
        Property SpAttack As Byte
        Property SpDefense As Byte
        Property Exp As Integer
        Property Name As String
        ReadOnly Property PokemonNames As Dictionary(Of Integer, String)
        ReadOnly Property LocationNames As Dictionary(Of Integer, String)
        Property Attack1 As IMDAttack
        Property Attack2 As IMDAttack
        Property Attack3 As IMDAttack
        Property Attack4 As IMDAttack
    End Interface

End Namespace

File: SkyEditor.SaveEditor/MysteryDungeon/Explorers/IExplorersStoredPokemon.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SkyEditor.SaveEditor.MysteryDungeon.Explorers
{
    public interface IExplorersStoredPokemon
    {
        int Level { get; set; }
        ExplorersPokemonId ID { get; set; }
        int MetAt { get; set; }
        int MetFloor { get; set; }
        int EvolvedAtLevel1 { get; set; }
        int EvolvedAtLevel2 { get; set; }
        int IQ { get; set; }
        int HP { get; set; }
        int Attack { get; set; }
        int Defense { get; set; }
        int SpAttack { get; set; }
        int SpDefense { get; set; }
        int Exp { get; set; }
        string Name { get; set; }
        int Tactic { get; set; }
        ExplorersAttack Attack1 { get; set; }
        ExplorersAttack Attack2 { get; set; }
        ExplorersAttack Attack3 { get; set; }
        ExplorersAttack Attack4 { get; set; }
    }
}
